README.md

```diff
@@ -112,39 +112,68 @@ Django Commons packages.
-   # ...
-   "repo-name" = {
-      description = "repo description"
-      allow_auto_merge = false # optional, default is false
-      allow_merge_commit = false # optional, default is false
-      allow_rebase_merge = false # optional, default is false
-      allow_squash_merge = false # optional, default is false
-      allow_update_branch = false # optional, default is false
-      enable_branch_protection = true # optional, default is true
-      has_discussions = true # optional, default is true
-      has_downloads = true # optional, default is true
-      has_wiki = false # optional, default is false
-      is_template = false # optional, default is false
-      push_allowances = []
-      required_status_checks_contexts = [] # optional, default is []
-      template = "" # optional, default is ""
-      topics = []
-      visibility = "public" # optional, default is "public"
-      skip_team_creation = false # Optional, default is false => create 3 teams for the repository
-      admins = [] # Members of the repository's admin and repository teams. Have admin permissions
-      committers = [] # Members of the repository's committers and repository teams. Have write permissions
-      members = [] # Members of the repository team. Have triage permissions
-   }
+  # ...
+  "repo-name" = {
+    description = "repo description"
+    homepage_url = "" # optional, default is ""
+    allow_auto_merge = false # optional, default is false
+    allow_merge_commit = false # optional, default is false
+    allow_rebase_merge = false # optional, default is false
+    allow_squash_merge = true # optional, default is true
+    allow_update_branch = true # optional, default is true
+    delete_branch_on_merge = true # optional, default is true
+    has_discussions = true # optional, default is true
+    has_downloads = true # optional, default is true
+    has_wiki = false # optional, default is false
+    is_template = false # optional, default is false
+    push_allowances = []
+    template = "" # optional, default is ""
+    topics = []
+    visibility = "public" # optional, default is "public"
+    skip_team_creation = optional(bool, false) # Do not create teams for repository
+    enable_branch_protection = true # optional, default is true
+    required_status_checks_contexts = [] # optional, default is []
+    admins = [] # Members of the repository's admin and repository teams. Have admin permissions
+    committers = [] # Members of the repository's committers and repository teams. Have write permissions
+    members = [] # Members of the repository team. Have triage permissions
+  }
 }
 ```
 
+2. import the repository into the terraform state by
+   running `terraform import -var-file=production/org.tfvars -var-file=production/repositories.tfvars -var github_token=... 'github_repository.this["repo-name"]' repo-name`
+3. Run `terraform plan -var-file=production/org.tfvars -var-file=production/repositories.tfvars -var github_token=...`
```

Comment:
Are steps 2 and 3 required? It seems like we should be able to use the PR process to create the plan changes and review them.

Reply:
Step 2 is mandatory, since it marks the repository as "managed by terraform". Step 3 is optional.

Reply:
Sounds good. Let's create an issue to automate that somehow to avoid needing a local terraform installation.